ivy/functional/backends/numpy/linear_algebra.py

```diff
@@ -97,6 +97,8 @@ def matrix_norm(
     out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
     ret = np.linalg.norm(x, ord=ord, axis=(-2, -1), keepdims=keepdims)
+    if not isinstance(ret, np.ndarray):
```

Comment:
we should use [this decorator](https://github.com/unifyai/ivy/blob/master/ivy/functional/backends/numpy/elementwise.py#L32) for this. Maybe you can move it into a new `helpers.py` instead, located at `ivy/functional/backends/numpy/helpers.py` rather than having it in `elementwise.py`, it's a common problem for numpy functions.

Reply:
Okay, that's great!